src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.cpp

```diff
@@ -143,9 +144,9 @@ tnsr::a<DataType, SpatialDim, Frame> gauge_source(
     const Scalar<DataType>& trace_extrinsic_curvature,
     const tnsr::i<DataType, SpatialDim, Frame>&
         trace_christoffel_last_indices) noexcept {
-  DataType one_over_lapse = 1. / get(lapse);
-  auto gauge_source_h =
+  *gauge_source_h =
```

Comment:
would `destructive_resize` work here? This current does a bunch of allocations even if the tensor is already allocated. You might have to zero the data after the `destructive_resize` call, but that's still cheaper than the allocations